src/test/java/uk/gov/companieshouse/company/metrics/kafka/AbstractKafkaTest.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.companieshouse.company.metrics.kafka;
+
+import org.springframework.context.annotation.Import;
+import org.testcontainers.containers.KafkaContainer;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+import org.testcontainers.utility.DockerImageName;
+import uk.gov.companieshouse.company.metrics.util.TestConfig;
+
+@Testcontainers
+@Import(TestConfig.class)
+public abstract class AbstractKafkaTest {
+
+    @Container
+    protected static final KafkaContainer kafka = new KafkaContainer(DockerImageName.parse(
+            "confluentinc/cp-kafka:latest"));
```

Comment:
Using the "latest" image maps to Kafka [version 7.7.0](https://hub.docker.com/layers/confluentinc/cp-kafka/7.7.0/images/sha256-76288746f518b38eb30496f5b8a9f157fedb0418bcca3648af903d03fc16d2d3?context=explore). We should use a specific version that is close to the version in Live